lib/jeff/device.ex

```diff
@@ -98,6 +123,18 @@ defmodule Jeff.Device do
     {device, command}
   end
 
+  def next_command(
+        %{
+          security?: true,
+          install_mode?: true,
+          secure_channel: %{established?: true, scbkd?: true},
+          address: address
+        } = device
+      ) do
+    command = Command.new(address, KEYSET, key: device.scbk)
+    {device, command}
+  end
```

Comment:
Where I got hung up on this PR was trying to decide how much install mode Jeff should be responsible for. This is the basic logic implemented in this PR in pseudocode:

```
if Jeff.add_device was called with a non-nil SCBK:
  Try to establish a secure channel with the given SCBK
  If secure channel is successful:
    Done
# Else if install mode is not allowed:
#   I was planning to implement this case, but I removed it
#   because of the same issue in the on failure case below
#   (not having a way to signal that a device is not working
#   properly)
  Else:
    Try to connect with the install mode SCBK
    On success:
       Issue a KEYSET command to set the SCBK on the device
       Restart the secure channel with the new SCBK
    On failure:
       TODO -- what _should_ happen here? We don't currently
       have a way to indicate that a device isn't working
       as expected
```